File: web/components/buttons/buttons-demo.ts
/// <reference path="../../../tsd.d.ts" />

import {
  Component, View,
  CORE_DIRECTIVES, FORM_DIRECTIVES
} from 'angular2/angular2';

import {ButtonCheckbox, ButtonRadio} from '../../../lib/index';

// webpack html imports
let template = require('./buttons-demo.html');

@Component({
  selector: 'buttons-demo'
})
@View({
  template:template,
  directives: [
    ButtonCheckbox, ButtonRadio,
    CORE_DIRECTIVES, FORM_DIRECTIVES
  ]
})
export class ButtonsDemo {
  private singleModel:string = '1';
  private radioModel:string = 'Middle';
  private checkModel:Object = {left: false, middle: true, right: false};
}

File: web/components/buttons/buttons-demo.dart
import "package:angular2/angular2.dart";
import "package:ng2_strap/buttons/button-checkbox.dart";
import 'package:ng2_strap/buttons/button-radio.dart';

@Component(
    selector: "buttons-demo",
    templateUrl: "buttons-demo.html",
    directives: const [
      ButtonCheckbox,
      ButtonRadio,
      CORE_DIRECTIVES,
      FORM_DIRECTIVES
    ])
class ButtonsDemo {
  String singleModel = "1";
  String radioModel = "Middle";
  Map checkModel = {"left": false, "middle": true, "right": false};
}
